concepts/basics/about.md

```diff
@@ -0,0 +1,3 @@
+# About
+
+[TODO]: Write about Whitespace
```

Comment:
That exactly. I just want to make it clear, that whitespace does not really matter, but is important for the human reader. When you come from Python, it might be strange to see "malformed" code.